website_sale_infinite_scroll/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module replaces the usual pagination in e-commerce
```

Comment:
This module removes pagination in /shop page and replaces it with infinite scrolling of products.